matrix-sdk/src/main/java/org/matrix/androidsdk/call/MXCallsManager.java

```diff
@@ -223,12 +225,10 @@ private IMXCall createCall(String callId) {
         }*/
 
         // webrtc
-        if (null == call) {
-            try {
-                call = new MXWebRtcCall(mSession, mContext, getTurnServer());
-            } catch (Exception e) {
-                Log.e(LOG_TAG, "createCall " + e.getMessage(), e);
-            }
+        try {
```

Comment:
I assume the code above will never be uncommented, else we should keep the `if` test.